Source/Urho3D/UI/UI.cpp

```diff
@@ -2076,16 +2078,31 @@ IntVector2 UI::GetEffectiveRootElementSize(bool applyScale) const
     return size;
 }
 
-void UI::SetRenderToTexture(UIComponent* component, bool enable)
+void UI::SetRenderTexture(UIElement* element, Texture2D* texture)

```

Comment:
Don't really like this name. Maybe, `SetElementRenderTexture` is more meaninful... What do you think?

Reply:
Sounds good to me